### ISSUE TITLE
No notification of someone leaving teleconference

### ISSUE THREAD

**Issue by norrinr** [wwivbbs/wwiv#1477]
When you are in the teleconference with someone, nothing tells you when someone has left or disconnected. This is with latest WWIV in Linux.

**Comment by wwiv** [comment]
Do you mean '/c' for multi user chat?

**Comment by norrinr** [comment]
Yes. I checked this again, but was only able to duplicate it when a user is in chat and just disconnects (doesn't type /q). There is no message to the other user of the signoff/disconnect.

**Comment by wwiv** [comment]
aha, thanks, I think that last bit of detail will help me figure out what's going on.  I'm glad to hear the multi-node chat is working for you since it needs a lot of attention I think.

**Comment by ericpareja** [comment]
confirmed that no disconnect message appears to others in the chat room in /c multi user chat when the other user simply disconnects/hangs up. the disconnected user is also still in the chat room.
there is a disconnect message if they /q)uit.
